src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BindingBuildTest.cs

```diff
@@ -59,12 +59,16 @@ public void CleanBasicBindingLibrary (string classParser)
 			using (var b = CreateDllBuilder (Path.Combine ("temp", TestName))) {
 				Assert.IsTrue (b.Build (proj), "Build should have succeeded.");
 				Assert.IsTrue (b.Clean (proj), "Clean should have succeeded");
-				var fileCount = Directory.GetFiles (Path.Combine (Root, b.ProjectDirectory, proj.IntermediateOutputPath), "*", SearchOption.AllDirectories)
-					.Where (x => !Path.GetFileName (x).StartsWith ("TemporaryGeneratedFile")).Count ();
+				var ignoreFiles = new string [] {
+					"TemporaryGeneratedFile",
+					"FileListAbsolute.txt",
```

Comment:
Why do we now need to ignore the `FileListAbsolute.txt` file? That should be deleted on Clean? If not its possibly an MSbuild Bug. //cc @radical

Reply:
I am not sure what should be expected here. I know I've had to ignore this in similar external tests for a _long_ time now, it started happening in these tests after the .NET Standard and Mono updates in this PR though.